include/ppx/application.h

```diff
@@ -301,6 +301,12 @@ struct ApplicationSettings
         uint32_t  numFramesInFlight = 1;
         uint32_t  pacedFrameRate    = 60;
 
+        // If `foveationMode` is not `FOVEATION_NONE`, then the pipeline
+        // targeting the application's RenderPass must use the same foveation
+        // mode (`GraphicsPipelineCreateInfo.foveationMode`).
+        grfx::FoveationMode        foveationMode        = grfx::FOVEATION_NONE;
```

Comment:
After talking with Aliya, this has been removed to avoid over-complicating the Application class.

This means apps will need to create their own RenderPasses to use foveation. This might be simplified by adding hooks for applications to customize the Swapchain/RenderPasses created by Application.